src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -98,9 +98,7 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
                             />
                             <div style={{marginLeft:"45px", marginTop:"5px", marginBottom:"10px"}}>
                                 <ButtonGroup className="pull-right">
-                                    <Button className="btn-default" disabled={this.props.store.pdbChainDataStore.allData.length === 0} onClick={this.toggle3dPanel}>
-                                        3D Structure »
-                                    </Button>
+                                    <Button className="btn-default" disabled={this.props.store.pdbChainDataStore.allData.length === 0} onClick={this.toggle3dPanel} dangerouslySetInnerHTML={{ __html:['3D Structure &','raquo;'].join('') }} />
```

Comment:
actually, how about replacing this character with something like this: http://fontawesome.io/icon/angle-double-right/